### DISCUSSION TITLE
Need help with error message

### DISCUSSION THREAD

**Discussion by MiT550** [tbnobody/OpenDTU#824] [Q&A]
Hi! Sorry, I don't have any experience with vscode. Trying to flash my ESP32, I keep getting the following error message:
U8x8lib.cpp .pio\libdeps\generic\U8g2\src
unused variable 'data' [Ln884, Col12]
Can anybody help me? What am I doing wrong? Thanks in advance.

**Comment by tbnobody** [comment]
Its a warning... just ignore it
